src/client/components/app/RecommendationsCard/index.tsx

```diff
@@ -32,7 +32,8 @@ const ItemCard = styled(Card)<{ onClick: any }>`
   display: flex;
   align-items: start;
   min-height: 30rem;
-  min-width: 32%;
+  // display three columns on larger screens
+  min-width: 31%;
```

Comment:
can just set width to 30%, dont need min + max with same numbers

Reply:
If I do this, the line cards fill to fit the available width. That's why I need the `max-width: 32%`
![image](https://user-images.githubusercontent.com/11972548/219214074-99471b96-502c-4d48-aa99-8a8f41375fc8.png)

Reply:
there's a couple ways to solve that. 

We should really just be using css grids at this point which is much more structured and would auto prevent this automatically